### ISSUE TITLE
Add the option to disable the offical player list.

### ISSUE THREAD

**Issue by andy013** [PazerOP/tf2_bot_detector#314]
Add an easy option to disable the official player list and just use your own personal list. Some people don't want to trust someone else to mark players. I think you can do this currently with an empty file but having it as an easy tick option in the tool would be very helpful. 

I think this would build trust in the tool and help to increase adoption amongst certain players.

**Comment by 2l47** [comment]
This would also make testing easier by enabling the isolation of marking to a specific list.

**Comment by 2l47** [comment]
In my experience, deleting the official lists led to them being recreated. If using an empty file doesn't work (I didn't try that), you can replace them with "dummy" lists like these:
https://milenko.ml/playerlist.official.json
https://milenko.ml/rules.official.json

**Comment by Kenajcrap** [comment]
> This would also make testing easier by enabling the isolation of marking to a specific list.
the tool already says which lists are marking a player if you hover over the marked player in the playerlist or check the tool's logs.

> In my experience, deleting the official lists led to them being recreated. If using an empty file doesn't work (I didn't try that), you can replace them with "dummy" lists like these:
> https://milenko.ml/playerlist.official.json
> https://milenko.ml/rules.official.json

Leaving the file blank will suffice, since the examples you mentioned include the line "update_url" that means that the owner of that domain can update the files at any moment to include steamid's (just like the official or third party lists do). This is expected behavour, but not if your objective is to create a dummy file.

I would like there to be a way to easily disable the default list just because this would lead to more trust in the tool's intentions from players

**Comment by 2l47** [comment]
> the tool already says which lists are marking a player if you hover over the marked player in the playerlist or check the tool's logs.

Yes, but list creators as well as users might benefit from only seeing players marked by their lists, for the sake of testing or otherwise. e.g. while making a new rule, seeing a new mark show up clearly is important.

> since the examples you mentioned include the line "update_url" that means that the owner of that domain can update the files at any moment to include steamids (just like the official or third party lists do). This is expected behavour, but not if your objective is to create a dummy file that does nothing

Thanks for pointing that out, I just removed it.

**Comment by ClusterConsultant** [comment]
The playerlist is still showing as having an update url. Also they don't even need to have contents. They could literally just be blank.

Also as a rules contributor, I've never had issues testing a rule with all playerlists I usually use present. The program is also pretty verbose when someone is marked. I'm not saying the usecase is nonexistent, just seems like a weird angle.

I'll probably make a guide for this or include an option in the installer rewrite (the odds increase the more different people ask about this) but the best solution remains with the umbrella idea of in application playerlist management. See #268

**Comment by 2l47** [comment]
> The playerlist is still showing as having an update url.

Refresh/clear cache? I don’t see one.

**Comment by Kenajcrap** [comment]
![image](https://user-images.githubusercontent.com/25385928/128538731-5d1c1ba9-57da-452b-81ad-441b8c949047.png)
https://milenko.ml/playerlist.official.json still has its `update_url`
dude, just leave the file blank until this gets implemented

**Comment by Tiagoquix** [comment]
Make the file empty and try marking it as read-only.
